### PR TITLE
Sdss 130 dataset pub sub

### PR DESCRIPTION
**Pull request by JackRimmerAND**
### Screenshots (if appropriate):

### CODE REVIEW

File: src/app/config/config.py [hunk 1]
```diff
@@ -131,6 +141,7 @@ def __init__(self):
         self.GOOGLE_APPLICATION_CREDENTIALS = get_value_from_env(
             "GOOGLE_APPLICATION_CREDENTIALS"
         )
+        self.DATASET_TOPIC_ID = get_value_from_env("DATASET_TOPIC_ID")
```
[the remark this answers is not in the document]
Comment (JackRimmerAND) [reply src/app/config/config.py]: Done

**Review by IsaiahLai111 — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -7,6 +7,7 @@ AUTODELETE_DATASET_BUCKET_FILE=True
 LOG_LEVEL=INFO
 PROJECT_ID = $(shell gcloud config get project)
 API_URL:=http://localhost:3000
+DATASET_TOPIC_ID=ons-sds-dataset-events
```
Comment (tidmanmike) [inline Makefile:10]: This should be more generic to the event please such as "ons-sds-publish-dataset-event"

Comment (JackRimmerAND) [reply Makefile:10]: Will update

Comment (JackRimmerAND) [reply Makefile]: @tidmanmike Updated this

**Review by Mohammed1804 — APPROVED** [review]

**Pull request merged**